scripts/capture_wiki_zhcn.py

```
@@ -0,0 +1,101 @@
+        except Exception as e:
+            print("init goto:", e, flush=True)
+        page.wait_for_timeout(8000)
+
+        for sub in SUBS:
+            url = (f"https://wiki.skland.com/endfield/catalog"
+                   f"?mainTypeId=1&typeSubId={sub}&filterIds=&header=0")
+            try:
+                with page.expect_response(
+                        lambda r, s=sub:
+                        r.url.startswith(PREFIX)
+                        and f"typeMainId=1" in r.url
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)